packages/patternfly-4/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -2,35 +2,115 @@
+
+Below is a group of icons next to a group of buttons. Note the cyan 24px spacer between them.
+
+![multiple groups](./img/multiple-groups.png)
+
+A single item that doesn't belong to a larger group is treated as a group on its own. It uses a 24px spacer on either side.
+
+Below is a single button next to a group of icons.
+
+![single item as group](./img/single-item-as-group.png)
+
+### Toggle groups
+A toggle group can be used when you want to collapse a set of items into an overlay panel at a certain breakpoint. This allows complex toolbars with multiple items and groups of items to be responsive. A toggle group is useful for containing filter controls, for example. When the toolbar responds to adapt to a mobile viewport, the contents contained in a toggle group will collapse into an overlay panel that can be toggled by clicking the Filter icon.
+
+![Toolbar with toggle group](./img/mobile-basic.png)
+
+![Toolbar with expanded toggle group](./img/mobile-expanded.png)
+
+### Overflow menus
+An overflow menu is a component that can be used in a toolbar (or elsewhere) to group a number of actions that should be collapsed into a kabob at a certain breakpoint. In the toolbar, the overflow menu will commonly be used to group a set of actions that are intended to collapse into the kabob at smaller viewport sizes.
+
+![toolbar with overflow menu](./img/basic-toolbar-nocallout.png)
+
+![toolbar with overflow collapsed](./img/mobile-basic-copy.png)
+
+In the above example, when the viewport size is reduced both the filter controls, contained in a toggle group, and the global actions, contained in an overflow menu, collapse to minimize the space required by the toolbar.
+
+### Vertical dividers
+When you have a large toolbar containing many groups of items, use a vertical divider to further
+separate the groups visually.
+
+![toolbar with divider](./img/toolbar-with-divider.png)
+
+## Common toolbar items and groups
+
+![common toolbar items](./img/toolbar.png)
+
+**1. Bulk selector:** A Bulk selector can be added when you want to give users the ability to select multiple items within a page. Besides allowing the user to select and deselect all items on a page, other application specific options can be added in the dropdown. Bulk selection allows the user to select or deselect all items on a page and provide an indeterminate state that notifies the user when only some items are selected. Bulk selectors should always be the left-most element in a toolbar. When providing a bulk selector in a list view, it should be aligned with the checkboxes.
+
+
+**2. Filter group:** PatternFly supports a variety of common filter types that can be used alone or in combination to construct a custom filter. You may consider placing filters inside of a toggle group to make you toolbar responsive. See Filters [link to the Filter documentation] for more information about available filters.
```

Comment:
Hey @mcarrano , this still says [link to the Filter documentation] - I'm unsure if that is intentional or not at this point

Reply:
Fixed